train.py

```diff
@@ -312,6 +326,7 @@ def train_func (data_path, existing_model, model_path, BATCH_SIZE, visualize_dat
 
         if not enable_regression:
```

Comment:
Now i know what you mean with including validation in training pipeline. 
I mean it can be also useful for developing and creating the right model but maybe at least make the whole validation process (starting from 327) optional?
Or for developing pipeline we create a complete new validation function seperated from training code.

Reply:
Do you believe it would improve the code to introduce a new boolean variable, such as 'Evaluate', in the 'params_and_main.py' file? This would allow the user to specify if evaluation should be included after the training pipeline by setting 'Evaluate = True'. Additionally, they could specify which data they want to evaluate from the list of valid scenes, such as ['vali'] or ['test'].

Reply:
I think this could make sense, the only risk I see is further bloating the params part. If so, I would try to find a descriptive name like 'save_confusion_matrix' or so.